surveys/2022-annual-survey/questions.md

```diff
@@ -298,60 +330,95 @@ Activities:
+
+Type: select one
+
+- Yes, for the majority of my coding
+- Yes, but I only use it occasionally
+- No (skip to `To what extent is Rust currently being used by your company?`)
 
 > **justification**
 >
-> We'd like to know where people are feeling unwelcome and the degree to which they are feeling unwelcome. This can 
-> help us better understand the free from responses that will come in the next question.
+> We want to establish what percentage of those who could possibly use Rust in a professional setting
+> are using Rust in a professional setting. This is most interesting over time.
+> Answers to this question should be combined with whether the respondent has ever used Rust.
 
-### If you indicated that you did not feel welcome in the Rust community, are there any details about your experience that you would like to share with us?
+
+### Which of the following statements are reasons why your team uses Rust at work
+
+Type: select all that apply
+
+Statements:
+
+- For its performance (i.e., speed, memory footprint, etc.) characteristics
+- We need precise control over exactly how our software runs
+- Its security and safety properties are important to us
+- It allows us to build relatively correct and bug free software
+- We find it enjoyable or fun to program in Rust
+- We already know Rust so it's our default choice
+- We find it easy to prototype with
+- We must interact with existing Rust code
+
+
+> **justification**
+>
+> The Rust community and potential adopters of Rust have a lot of assumptions of why one would choose Rust for a project.
+> This question can help confirm or challenge our assumptions and see how they change over time.
+
+
+### Which of the following statements apply to your team's experience using Rust at work.
+
+Type: select all that apply
+
+Statements:
+
+- Using Rust has helped us achieve our goals
+- Adopting Rust has been challenging
+- Overall, adopting Rust has slowed down our team
+- Using Rust has been worth the cost of adoption
+- We're likely to use Rust again in the future
+
+> **justification**
+>
+> Future potential adopters may want to know how often other's have encountered success.
+
+### What about your usage of Rust has been challenging?
 
 Type: free form (optional)
 
 > **justification**
-> 
-> More detail on the type of situations where people have felt unwelcome can let us better 
-> address these issues in the future.
+>
+> This an opportunity to learn from adopters at companies what they struggle with when adopting Rust.
 
-## Rust at work
 
 ### To what extent is Rust currently being used by your company?
 
 Type: select one
 
-- My company uses Rust for a large portion of production projects
-- My company uses Rust for a small portion of production projects
-- My company uses Rust only for non-production projects (e.g., tooling)
-- My company has actively experimented with Rust
-- My company has seriously considered, but not experimented with, using Rust
+- My company uses Rust
```

Comment:
It would be nice if we could still get some signal between "my company uses Rust but it's not an important vs. unimportant piece of our tech stake"